web/src/controller/agreement_controller.rs

```diff
@@ -139,15 +139,12 @@ pub async fn index(
     // TODO: create a new Extractor to authorize the user to access
     // the data requested
     State(app_state): State<AppState>,
-    Query(params): Query<HashMap<String, String>>,
+    // params: Result<Query<IndexParams>, QueryRejection>,
```

Comment:
@calebbourg Is this needed as documentation?

Reply:
Nope :) updated